app/generator/content/part2/map-reductions.js

```diff
@@ -0,0 +1,20 @@
+const mapReductions = (funding) => {
+  const reductions = []
+  let i = 1
+  funding.forEach(x => {
+    if (x.reductions) {
+      x.reductions.forEach(y => {
+        if (!reductions.find(z => z.reason === y.reason)) {
```

Comment:
suggestion: personally, I think `!==` inside reads easier rather than `===` and negating the whole thing

Reply:
It wouldn't work with !== as then it would return as undefined for first item and then defined for second when no match.  Could use `some` rather than `find` if that's clearer?